staging/vhost-device-sound/src/device.rs

```diff
@@ -681,7 +681,7 @@ impl VhostUserSoundBackend {
             threads,
             virtio_cfg: VirtioSoundConfig {
                 jacks: 0.into(),
-                streams: 1.into(),
+                streams: Le32::from(streams_no as u32),
```

Comment:
This is a pre-existing issue, right?
I mean, it's true that we didn't support RX yet, but we already had 2 streams allocated.

Maybe better to put in another patch (not a strong opinion)

Reply:
The issue was that returning `Le32::from(streams_no as u32)` would show a capture stream to the guest but it wouldn't work at all. Now it will work depending on backend support. IMO we should change the VirtioSoundConfig value to runtime computed values depending on the audio backend, since it has to query the host for what streams are available, what channel maps are available, etc. We should do that in a separate PR.

Reply:
Ack, I agree!